Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+
+Nova Conductor services, called "Super Conductor", continue to run on the main controller nodes.
+
+The services in the Cell Controllers can still call placement APIs, but cannot access other API layer services via RPC, nor can they access global API databases on the control nodes.
+
+## Definition of a Control Plane
+
+In Openstack/SCS, a Control Plane consists of 5 hardware nodes, which together serve several Availability Zones and thus provide a common usable API for a region. The Control Plane also shares the network (Neutron), the Sheduler and the CEPH services.
+
+It includes the Controller Nodes (Galera Cluster, RabbitMQ) and the Manager Nodes, Maas,...
+
+## Definition of Control Node
+
+The Control Node runs the Identity Service, Image Service , management processes for compute nodes, management processes for networking, various networking agents, and the Dashboard. It also includes supporting services such as an SQL database, a message queue, and NTP.
+
+Optionally, the Controller Node runs parts of the Block Storage, Object Storage, Orchestration and Telemetry services.
+
+The Controller Node requires at least two network interfaces.
+
+## Definition of Compute Node
+
+A compute host runs the hypervisor part of compute that runs instances. By default, compute uses the KVM hypervisor. The compute host also runs a networking service agent that connects instances to virtual networks and provides firewall services to the instances through security groups.
+
+If you offer hyper-converged infrastructure, a compute host also serves the Ceph. This makes the storage dynamically scalable (horizontally and vertically). For the Ceph services, 1 CPU core and 4 GB of RAM are reserved per OSD to ensure appropriate performance. 
+
+## Definition of Manager Node
+
+From here, the OSISM Ansible playbooks are applied to the environment. Furthermore, the following services often run here non-redundantly: Prometheus server,....
+
+## Definition of provider network
+
+The provider network is the network that is "in front", i.e. at the output points of the openstack/SCS. This is usually a public network, but can also be a private network in individual cases. IPs from the provider network can be assigned to instances within the SCS. The same applies to load balancers, of course. 
+
+## Definition of API
+
+The Rest API provides the core of openstack/SCS  and can be addressed for a whole region. It accepts and responds to end-user API calls.  The service supports the OpenStack Compute API, the Amazon EC2 API, and a special Admin API for privileged users to perform administrative actions. Policies are enforced and most orchestration actions can be started, such as launching an instance.
+
+## Horizon
+
+Horizon is openstack's preferred GUI for the end user, but also for the administrator for a quick overview. It runs on the controller node. 
+
```

Comment:
Please define "preferred". I am not sure it's the case for all the vendors. (See also new projects around dashboards)

Reply:
Especially with skyline being announced and it's in active development now https://opendev.org/skyline/skyline-apiserver

Reply:
Horizon is still the "reference" implementation and actively developed by OpenStack. Skyline is the new approach which is currently still not the "preferred" method.